OVERVIEW.md

```diff
@@ -75,61 +112,82 @@ You might want to use this library and contribute to development, if any of the
+|       Elastic Search       |   elastic_ecs    |    ECS     | IBM Security |     Yes     |     Yes      |   Release    |
+| IBM Cloud Security Advisor | security_advisor |  Default   |  IBM Cloud   |     Yes     |     Yes      |   Release    |
+|           Splunk           |      splunk      | Splunk CIM | IBM Security |     Yes     |     Yes      |   Release    |
+|       Microsoft ATP        |      msatp       |  Default   | IBM Security |     Yes     |     Yes      |   Release    |
+|        IBM Guardium        |     guardium     |  Default   | IBM Security |     Yes     |     Yes      |   Release    |
+|    AWS CloudWatch Logs     |                  |  Default   | IBM Security |     Yes     |     Yes      |   Planned    |
+|       Azure Sentinel       |                  |  Default   | IBM Security |     Yes     |     Yes      |   Planned    |
+
+The full list of connectors and their supported STIX objects and properties can be found [in this table](https://ibm.box.com/v/STIX-object-support).
 
 ## How to use
 
-Stix-shifter handles two primary functions:
-
-1. **Translation**
-   Stix-shifter translates STIX patterns into data source queries (in whatever query language the data source might use) and from data source results into bundled STIX observation objects (very similar to JSON).
-2. **Transmission**
-   Passes in authentication credentials to connect to a data source where stix-shifter can then ping or query the data source or fetch the query status and results.
+### Prerequisites
 
 Python 3.6 is required to use stix-shifter.
 
-### Translation
+Stix-shifter provides several functions: `translate` and `transmit` are the primary functions, `execute` offers a way to test the complete stix-shifter flow.
+
+1. [**Translate**](#translate)
 
-#### Translate a STIX 2 pattern to a native data source query
+   The `translate` command converts STIX patterns into data source queries (in whatever query language the data source might use) and translates data source results (in JSON format) into bundled STIX observation objects.
 
-##### INPUT: STIX 2 pattern
+2. [**Transmit**](#transmit)
+
+   The `transmit` command allows stix-shifter to connect with products that house repositories of cybersecurity data. Connection and authentication credentials are passed to the data source APIs where stix-shifter can make calls to ping the data source, make queries, delete queries, check query status, and fetch query results.
+
+3. [**Execute**](#execute)
+
+   The translation and transmission functions can work in sequence by using the `execute` command from the CLI.
+
+## Translate
+
+### 1. Translate a STIX pattern to a native data source query
+
+#### INPUT: STIX 2 pattern
 
 ```
 # STIX Pattern:
 "[url:value = 'http://www.testaddress.com'] OR [ipv4-addr:value = '192.168.122.84']"
 ```
 
-##### OUTPUT: Native data source query
+#### OUTPUT: Native data source query
 
 ```
-# Translated Query:
+# Translated Query (using SQL as an example):
```

Comment:
would this be AQL?

Reply:
They would be the same. I'm trying to keep the examples generic where possible. This does remind me though that I wanted to clarify the comments in the dummy query _constructo.